git-filter-repo

```diff
@@ -873,15 +873,15 @@ class Alias(_GitElement):
 
   def __init__(self, ref, to_ref):
     _GitElement.__init__(self)
-    # Denote that this is a reset
+    # Denote that this is a alias
```

Comment:
an alias

Reply:
@marcows Thanks for that catch! LOL, I'll trade you a typo for a typo. I was also reviewing more closely branch names from contributors, which convention I did not follow. I am going to open a new pull request from a different branch. Thanks for your patience!

Reply:
The branch naming like zs/xxx is used by Elijah locally (visible to us because of local merge without the GitHub merge button), it is not a convention for you to follow. And even if it was, you just should have updated your branch instead of creating a new PR.

Reply:
@marcows Thanks for taking the time to correct me here. I'll see if I can do it right. I applied your correction in the original branch, but perhaps made a further mistake by doing an interactive rebase to squash the fix-up with the original commit attempting to fix the "alias" typo. I hope that wasn't another no-no.